sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -114,6 +114,8 @@ public class EventHubProducerClient : IAsyncDisposable
         ///   Event Hub will result in a connection string that contains the name.
         /// </remarks>
         ///
+        /// <seealso href="https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string"/>
```

Comment:
Thanks for adding these everywhere!

Reply:
Thanks to @christothes for that one.   He called it out.   :smile: